hw/ip/otbn/doc/dv/index.md

```diff
@@ -136,12 +141,18 @@ That is, we expect to see execution with each bit of each immediate field being
 We also expect to see each field with values `'0` and `'1` (all zeros and all ones).
 If the field is treated as a signed number, we also expect to see it with the extremal values for its range (just the MSB set, for the most negative value; all but the MSB set, for the most positive value).
 
-For any instruction that reads from or writes to a GPR register, we expect to see that operand equal to `x0`, `x1` and an arbitrary register in the range `x2 .. x31`.
+For any instruction that reads from or writes to a GPR, we expect to see that operand equal to `x0`, `x1` and an arbitrary register in the range `x2 .. x31`.
+We don't have any particular coverage requirements for WDRs (since all of them work essentially the same).
 
 For any source GPR, we require "toggle coverage" for its value.
 For example, `ADD` reads from its `<grs1>` operand.
 We want to see each of the 32 bits of that operand set and unset (giving 64 coverage points).
 
+If an instruction can generate flag changes, we expect to see each flag that the instruction can change being both set and cleared by the instruction.
+This needn't be crossed with the two flag groups (that's tracked separately in the "Flags" block above).
+For example, `BN.ADD` can write to each of the flags `C`, `M`, `L` and `Z`.
+This paragraph implies eight coverage points (four flags times two values) for that instruction.
+
 #### ADD
 
 - Each of the 11 interesting sign behaviours:
```

Comment:
I'm now wondering if we need all the various positive plus negative etc points for `ADD` and `SUB`. There's no difference between signed and unsigned numbers for the operation so wouldn't an `ADD` or `SUB` seeing overflow and seeing no overflow suffice?

Reply:
Yes, I think I agree. Dropping them and replacing with the obvious cross: (neg / zero / pos)^2. This will give the overflow checks on the side.